mandatory/3-playing-computer.js

```diff
@@ -6,13 +6,13 @@
 
   Answer the following questions:
 
-  1. This program throws an error. Why? (If you can't find it, try executing it).
+  1. This program throws an error. Why? (If you can't find it, try executing it). -  b is not defined
   2. Remove the line that throws the error.
   3. What is printed to the console?
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
-  7. What is the value of the "a" outer variable when "f1" is called for the first time?
+  4. How many times is "f1" called? - 3 times
+  5. How many times is "f2" called? - 2 times
+  6. What value does the "a" parameter take in the first "f1" call?  a = 7
```

Comment:
it's 1 because we put i for a parameter in the function so that time our i is equal to 1